libdnf/module/ModulePackageContainer.hpp

```diff
@@ -107,6 +107,16 @@ struct ModulePackageContainer
 
     std::string getReport();
 
+    /**
+     * @brief Get configured default stream for a module
+     */
+    const std::string & getDefaultStream(const std::string &name);
```

Comment:
No, not in Fedora

Reply:
Per the modulemd specification, a "default" profile is assumed to _always_ exist--even if it isn't explicitly defined. See https://github.com/fedora-modularity/libmodulemd/blob/master/spec.v2.yaml#L154 and https://bugzilla.redhat.com/show_bug.cgi?id=1568165

Reply:
Well try it with F28

Reply:
It doesn't work because it's a bug that needs to be fixed or because F28 has an older version of libmodulemd' specification?

Reply:
Please can you add information about possibility of throwing exception (std::out_of_range) in case that name was not found? This is a required style of the project. Thanks a lot.